gradio/external.py

```diff
@@ -33,6 +31,45 @@
     from gradio.interface import Interface
 
 
+set_documentation_group("helpers")
```

Comment:
nit: I think the way `set_documentation_droup` works, it doesn't really allow calling it twice because it resets the classes to document for that group every time it's called. 

Just tested locally and this is not showing up. 

Let's fix before next release!

Reply:
Not the first time that has tripped me up 🤦 Disregard!